etcdctl/ctlv3/command/lease_command.go

```diff
@@ -134,3 +135,33 @@ func leaseKeepAliveCommandFunc(cmd *cobra.Command, args []string) {
 	}
 	fmt.Printf("lease %016x expired or revoked.\n", id)
 }
+
+// NewLeaseProbeCommand returns the cobra command for "lease probe".
+func NewLeaseProbeCommand() *cobra.Command {
+	lc := &cobra.Command{
+		Use:   "probe <leaseID>",
+		Short: "Get lease information",
+
+		Run: leaseProbeCommandFunc,
+	}
+
+	return lc
+}
+
+// leaseProbeCommandFunc executes the "lease probe" command.
+func leaseProbeCommandFunc(cmd *cobra.Command, args []string) {
+	if len(args) != 1 {
+		ExitWithError(ExitBadArgs, fmt.Errorf("lease probe command needs lease ID as argument"))
+	}
+
+	id, err := strconv.ParseInt(args[0], 16, 64)
+	if err != nil {
+		ExitWithError(ExitBadArgs, fmt.Errorf("bad lease ID arg (%v), expecting ID in Hex", err))
+	}
```

Comment:
this path happens several times in `lease_command.go`; have something like `leaseFromArgs(args []string) v3.LeaseID` that handles all the error checking / exiting for probe/renew/revoke?